README.md

```diff
@@ -1 +1,157 @@
+python manage.py form 20221201 user R149.1
+
+# The CI links to a list of HGNC IDs
+python manage.py form 20221201 user R417.2
+
+# The CI links to nothing
+python manage.py form 20221201 user R413.1
+```
+
+This command requires 4 arguments, the first 3 of which should come from the user request:
+- Request date (in YYYYMMDD format)
+- Requester initials
+- The clinical indication R code
+- The path to a text file dump of the HGNC database
+
+To create the HGNC text file:
+- Go to https://www.genenames.org/download/custom/
+- In the first section, uncheck boxes so that only 'HGNC ID', 'Approved symbol', 'Previous symbols' and 'Alias symbols' are left ticked
+- Scroll down and click 'submit'
+- Copy the entire resulting text output (ctrl + A), and paste into a text editor
+- Save as '<YYYYMMDD>_hgnc_dump.txt' in the same folder as the manage.py script
+
+Executing the command will generate an MS Excel file named 'request_form_<req_date>_<ci_code>_<requester>.xlsx' in the same folder as the manage.py script.
+
+The file consists of four pandas dataframes:
+1. Generic information about the request (request date, clinical indication used, genome build used etc.)
+2. Data on which panels are currently associated with that clinical indication
+3. A list of the genes in these panels and their associated metadata (mode of inheritance, penetrance etc.)
+4. A list of the regions in these panels and their associated metadata
+
+More information about each metadata field can be found in the PanelApp handbook at https://panelapp.genomicsengland.co.uk/media/files/PanelApp_Handbook_V18_120210506.pdf
+
+## Import a completed panel request form and update models
+The generic command for this is:
+``` 
+python manage.py seed form <fp>
+```
+- Example usage:
+``` 
+python manage.py seed form request_form_20221201_R149.1_user.xlsx
+```
+
+The command takes 1 argument (fp) which is the path to the completed request form.
+
+Executing the command reads in and parses a request form, creates a new panel instance from the request, links the relevant clinical indication to the new panel, and updates the link to the previous panel so that it is no longer current.
+
+## Running unit tests
+Unit tests are defined in requests_app/tests.py, and can be run from within the top-level panel_requests directory.
+
+- Example usages:
+```sh
+# Basic execution
+pytest
+
+# Output is quiet or verbose
+pytest -q
+pytest -v
+
+# Output includes line coverage
+pytest --cov=panel_requests panel_requests/requests_app/tests.py
+```
+## Generate genepanel
+```
+Make a HGNC dump txt file here: https://www.genenames.org/download/custom/
+Include columns:
+- HGNC id
+- Locus Type
+- Approved Name
+
+python manage.py genepanels --hgnc <hgnc dump path>
+python manage.py generate genepanels --hgnc testing_files/hgnc_dump_23052023.txt --output /home/jason/github/eris/testing_files
+```
+
+## Generate g2t
+This command required 5 files:
```

Comment:
this will need a lot more explanation, feel free to look into the panel manual for that